espresso/docker-compose.yml

```diff
@@ -35,18 +35,38 @@ services:
     ports:
       - "8545:8545" # L1 RPC
 
+  op-geth-init:
+    image: us-docker.pkg.dev/oplabs-tools-artifacts/images/op-geth:v1.101503.2-rc.3
+    volumes:
+      - ../config:/config
+      - op-geth-data:/data
+    entrypoint: ""
+    command: >
+      sh -c "
```

Comment:
This _might_ be really annoying in terraform. I'd recommend checking how arbitrary scripts work in there. Ideally we'd just make this part of the container instead, so we can fork their container and then add this as a setup step (I think).

Reply:
In the terraform I created in the `tee-op-deploy` repo, I added commands like this to the container definition. Can we keep them here till the terraform PR (not sent yet) is merged?